help/opers/snomask

```diff
@@ -9,11 +12,14 @@ Server notice masks:
        +C     - Extended local client connections and exits
        +d     - Server debug messages
        +f     - Clients refused due to limits
+    ?  +F     - Remote client connections and exits
        +k     - Server and service kill messages
+    ?  +l     - Channel creation messages
        +n     - Local client nick changes
```

Comment:
As per the channel discussion, this could be extended to also say `(and possibly global)` (if the admin has sno_globalnickchange) loaded.